### PR TITLE
LPS-146828 Add parameter to avoid rendering the link to the styles where we don't need it

### PR DESCRIPTION
**Pull request by victorg1991**
Hey @Tim-Cao I didn't manage to reproduce these two issues:
- https://github.com/liferay-echo/liferay-portal/pull/8520#issuecomment-1158384560
- https://github.com/liferay-echo/liferay-portal/pull/8520#issuecomment-1158458688

The others should be fixed with this PR :)

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:echo

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java [hunk 1]
```diff
@@ -448,8 +448,11 @@ public Map<String, Object> getEditorContext(String npmResolvedPackageName)
 					String layoutURL = PortalUtil.getLayoutFriendlyURL(
 						themeDisplay.getLayout(), themeDisplay);
 
-					return HttpComponentsUtil.addParameter(
+					layoutURL = HttpComponentsUtil.addParameter(
 						layoutURL, "p_l_mode", Constants.PREVIEW);
+
+					return HttpComponentsUtil.addParameter(
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java:454]: @victorg1991 🤔 Is it necessary?

**Comment by Tim-Cao** [comment]
Hi @victorg1991, there is a regression issue. The user cannot edit the inline text at Content Editing mode.
**Step to reproduce**
1. Add a Heading fragment to a content page
2. Switch to the Content Editing
3. Double click the body of Heading fragment

**Expected Results:**
The user should access to editiong of Heading.

**Actual Results:**
Nothing happens.

**Comment by Tim-Cao** [comment]
Hi @victorg1991, there is another regression issue.
The user only with Update - Limited permission cannot add a fragment to page
**Step to reproduce**
1. Add a site role named page builder with Update - Limited permission
2 .Add a new user
3 .Add a new site
4. Assign the new user to site
5. Assign the new site role to user
6. Add a content page in new site
7. Sign in as new user
8. Navigate to the page editor

**Expected Results:**
The Fragments and Widgets,Page Design Options should be shown.

**Actual Results:**
The Fragments and Widgets,Page Design Options are invisible.

**Comment by Tim-Cao** [comment]
Hi @victorg1991, there is another regression issue.
The topper of fragment is shown to user only with Update Page Content permission
**Step to reproduce**
1. Add a site role named content editor with Update Page Content permission
2. Add a new user
3. Add a new site
4. Assign the new user to site
5. Assign the new site role to user
6. Add a Heading fragment to a content page
7. Sign in as new user
8. Navigate to the page editor
9. Click the body of Heading fragment.

**Expected Results:**
The topper of Heading shouldn't be shown.

**Actual Results:**
The topper of Heading is shown. Then the user could remove the fragment.

**Comment by Tim-Cao** [comment]
Resent https://github.com/liferay-echo/liferay-portal/pull/8540

**Comment by victorg1991** [comment]
resent here: https://github.com/liferay-echo/liferay-portal/pull/8541

**Pull request closed without merging**